app/models/album.rb

```diff
@@ -1,6 +1,6 @@
 class Album < ActiveRecord::Base
-  validates :title, :genre, :artist, :length, acceptance: true
-  validates :title, :genre, :artist, :length, length: { minimum: 2 }
-  validates :title, uniqueness: { scope: artist }
-  validates :number, numericality { only_integer: true }
+  validates :title, :genre, :artist, :length, presence: true
+  validates :title, :genre, :artist, :length, length: { minimum: 1 }
```

Comment:
I think we spoke about this in class, but a minimum of 1 is the same test as `presence: true`, so this line isn't needed.